inc/saiswitch.h

```diff
@@ -472,6 +472,17 @@ typedef enum _sai_switch_attr_t
     SAI_SWITCH_ATTR_DEFAULT_VIRTUAL_ROUTER_ID,
 
     /**
+     * @brief Default .1Q Bridge ID
```

Comment:
This is change of behavior. In the current APIs all the ports are by default added to vlan 1. Please help me understand this.